build/package.js

```diff
@@ -21,18 +21,20 @@ const platforms = [
   "darwin-arm64",
 ];
 
+mkdirSync(resolve(__dirname, "..", "out", "vsix"), { recursive: true });
+
 // We include different files for each platform, so we need to build a custom
 // .vscodeignore file based on common file and platform-specific path.
 const ignoreFile = readFileSync(resolve(__dirname, "..", ".vscodeignore"), "utf8");
 const customIgnoreFilePath = resolve(__dirname, "..", "out", ".vscodeignore");
 
 execSync('npm run build', { cwd: resolve(__dirname, ".."), stdio: "inherit" });
-mkdirSync(resolve(__dirname, "..", "out", "vsix"), { recursive: true });
 for (const platform of platforms) {
   writeFileSync(
     customIgnoreFilePath,
     ignoreFile + `!assets/platform/${platform}/**`
   );
+  mkdirSync(resolve(__dirname, "..", "out", "vsix"), { recursive: true });
```

Comment:
Why the output directory needs to be created inside of the loop, all VSIX packages are written to the same directory, I think that it is enough to create the directory once before the for loop.